packages/rest/src/resource.ts

```diff
@@ -116,16 +116,19 @@ function cloneKeyProperties(context: DecoratorContext, target: Model, resourceTy
       // filtering. NOTE: Check for name equality instead of function equality
       // to deal with multiple copies of core being used.
       ...keyProperty.decorators.filter((d) => d.decorator.name !== $visibility.name),
-      {
-        decorator: $path,
-        args: [],
-      },
       {
         decorator: $resourceTypeForKeyParam,
         args: [{ node: target.node, value: resourceType, jsValue: resourceType }],
       },
     ];
 
+    if (!keyProperty.decorators.some((d) => d.decorator.name === $path.name)) {
```

Comment:
Ah - so was the issue that there were 2 `@path` decorators assigned if an augment was used, and the hard-coded one was being picked up?

Reply:
yes!! 😄  we were hard coding the one without any information and just assigning the path, replacing the value of the one that was passed, when it was passed